### DISCUSSION TITLE
Why does MUI textField not display error help text?

### DISCUSSION THREAD

**Discussion by Yedidya10** [react-hook-form/react-hook-form#10712] [Q&A]
I can't solve the following issue:  Why does the MUI text field element not display error help text?

https://codesandbox.io/s/react-hook-form-v7-controller-rules-and-helpertext-issue-93hkmq?file=/src/Mui.js

**Comment by Sajarin-M** [comment]
![image](https://github.com/react-hook-form/react-hook-form/assets/68892162/2b9473d5-41c7-4b39-b90e-b04f89ef9ed0)
Its displaying in your example

**Reply by Yedidya10** [reply]
Very strange. I tried in 3 browsers and it doesn't show an error message when I enter a false regex.
